### PR TITLE
create pull request template

### PR DESCRIPTION
**Pull request by dndrks**
@ngwese + @pq, does this work as a PR template for user contributions?

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+```
+{
+      "project_name": "example",
+      "project_url": "https://github.com/your-github/example",
+      "author": "Your Name",
+      "description": "a quick description to summarize functionality",
+      "discussion_url": "https://llllllll.co/t/example/000000",
+      "tags": ["example", "example", "example"],
+      "origin": "(if applicable) lines"
```
Comment (ngwese) [inline pull_request_template.md]: The `origin` property is a leftover from the lines forum scraper and not really needed. It is probably just as well that we remove it.

**Comment by dndrks** [comment]
@ngwese + @pq : lost mega track of this, apologies!

so, we've had 4 successful PR's into the repo -- seems to be going well without a template, for this who are already utilizing GitHub without trouble.

for new-comers, though, would it perhaps be interesting to add an **Issue** template that walks them through @pq 's excellent checkpoints? so then someone would just go to Issues + they'll see an "add my app" option?

**Comment by pq** [comment]
no worries at all.

i'd say it's entirely up to you whether this would add value or not.  my experience with templates has been a bit mixed to be honest.

maybe just keep it in mind and add it if the need feels acute down the road?

**Comment by pq** [comment]
oh wait, @dndrks, i just re-read your comment.

>  for new-comers, though, would it perhaps be interesting to add an Issue template that walks them through @pq 's excellent checkpoints? so then someone would just go to Issues + they'll see an "add my app" option?

👍 

SGTM!

**Pull request merged**